frontend/src/components/Collection/components/CollectionDatasetsGrid/components/Row/DownloadDataset/index.tsx

```diff
@@ -44,35 +39,19 @@ const DownloadDataset: FC<Props> = ({
     <>
       <Button
         datasetName={name}
-        disabled={isDisabled}
-        onClick={toggleOpen}
         data-testid="dataset-download-button"
+        disabled={isDisabled}
+        onClick={() => setIsOpen(true)}
       />
-      <Modal
-        title="Download Dataset"
-        isCloseButtonShown={false}
-        isOpen={isOpen}
-        onClose={toggleOpen}
-        className={isLoading || isError ? "modal-loading" : undefined}
-      >
-        {isLoading ? (
-          <ModalContentWrapper>
-            <Spinner size={20} />
-          </ModalContentWrapper>
-        ) : null}
-        {isError ? (
-          <ModalContentWrapper>
-            Dataset download is currently not available.
-          </ModalContentWrapper>
-        ) : null}
-        {!isLoading && !isError ? (
-          <Content
-            name={name}
-            dataAssets={datasetAssets}
-            onClose={toggleOpen}
-          />
-        ) : null}
-      </Modal>
+      <Dialog onClose={closeModal} open={isOpen}>
+        <Content
```

Comment:
Updating from `blueprintjs` `Modal` to `czi-sds` `Dialog` to achieve styles more closely aligned with mocks.